static/app/routes.tsx

```diff
@@ -43,10 +43,10 @@ const IndexRoute = BaseIndexRoute as React.ComponentClass<IndexRouteProps & Cust
 
 const hook = (name: HookName) => HookStore.get(name).map(cb => cb());
 
-const SafeLazyLoad = errorHandler(LazyLoad);
+export const SafeLazyLoad = errorHandler(LazyLoad);
 
-type PromisedImport<C> = Promise<{default: C}>;
-type ComponentType = React.ComponentType<any>;
+export type PromisedImport<C> = Promise<{default: C}>;
+export type ComponentType = React.ComponentType<any>;
```

Comment:
I don't really want to export any of these, especially not `ComponentType`.

Reply:
It just pollutes the auto-import namespace.

Reply:
@evanpurkhiser So should I just recreate the types in each file?

Reply:
yeah it's fine, I already removed some of these anyway